swiftype_app_search/client.py

```diff
@@ -88,7 +88,7 @@ def list_engines(self, current=1, size=20):
         name of the engine.
         """
         data = { 'page': { 'current': current, 'size': size } }
```

Comment:
`{ 'page' = 'blah' }` is also against PEP8 style @tehsven 
https://www.python.org/dev/peps/pep-0008/#pet-peeves

I don't quite regard to it as the bible, but this repository does have no spaces immediately after a parens or curly bracket